app/views/account_reset/confirm_request/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <% title t('account_reset.confirm_request.check_your_email') %>
 
 <div class="mtn2">
-  <%= image_tag asset_url('check-email.svg'), width: 24, class: 'left-0 right-0 ico-heading' %>
+  <%= image_tag asset_url('check-email.svg'), alt: '', width: 24, class: 'left-0 right-0 ico-heading' %>
```

Comment:
should we give it a real `alt` as long as we're here? Like "image of letter" ?

Reply:
Oh there are a lot of these.... maybe should lsit them and make a TODO/chore ticket to follow up

Reply:
We can, but I'm pretty sure I got this right.

Something we learned from a recent 508 audit is if an image is decorative the screen reader will read the same thing twice. Think "Check you email. Check email image." If you set the alt tag to an empty string the screen reader will skip decorative images.

Reply:
Ref: https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
Ok! That makes sense then